src/Weather.js

```diff
@@ -0,0 +1,24 @@
+class Weather {
+
```

Comment:
GOOD 3: This class was good addition as it contains logic that was needed for the project but wouldn't be cohesive in the other classes. 
BAD 3: My plan was to use `enumerators` (I think) for the weather states as they are something I only want to refer to, not change, though I don't think this was the best way to achieve this.
GOALS 3: Carry on with splitting up modules and code into their cohesiveness relative to eachother, learn the correct practice for making permanent variables.